content/reference/admin/custom_sso/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Custom SSO"
+description: "Learn how to set up a custom Single Sign-On (SSO) system for your organization."
+lead: "A custom SSO configuration allows your users to sign into Gatling Enterprise using your organization's authentication system."
+date: 2022-03-01T14:00:00+00:00
+lastmod: 2022-03-01T14:00:00+00:00
+weight: 22040
+---
+
+Configuring a custom Single Sign-On (SSO) solution is only available on [select pricing tiers](https://gatling.io/pricing/)
```

Comment:
```suggestion
Configuring a custom Single Sign-On (SSO) solution is only available on [corporate plan](https://gatling.io/pricing/)
```

Reply:
I didn't really want to duplicate pricing model info here, so we don't have to republish the doc when the pricing model changes (although in practice this one won't change while we require some manual setup...). But I should probably at least use "pricing plan" rather than "pricing tier" for consistency.

Reply:
Let's go with Corporate plan.